Kickstarter-iOS/Views/Controllers/ProjectPageViewController.swift

```diff
@@ -80,14 +80,17 @@ public final class ProjectPageViewController: UIViewController, MessageBannerVie
       |> \.dataSource .~ self.dataSource
       |> \.delegate .~ self
       |> \.tableHeaderView .~ self.projectNavigationSelectorView
-      |> \.tableFooterView .~ nil
+      |> \.tableFooterView .~ UIView(frame: .zero)
```

Comment:
Just realized setting to it `nil` is the same as the behavior whereas setting it to a blank view is what gets rid of the lines in the tableview.